tools/dune_rule_gen/coq_rules.ml

```diff
@@ -123,11 +123,14 @@ module Theory = struct
     (** Coq's logical path *)
     ; implicit : bool
     (** Use -R or -Q *)
+    ; deps : string list;
+    (** Adds as -Q user-contrib/X X *)
     }
 
-  let args { directory; dirname; implicit } =
+  let args { directory; dirname; implicit; deps } =
     let barg = if implicit then "-R" else "-Q" in
     Arg.[ A barg; Path directory; A (String.concat "." dirname) ]
+    @ List.flatten (deps |> List.map (fun dep -> Arg.[A "-Q"; A ("user-contrib/"^dep); A dep]))
```

Comment:
Should the middle one be Path?

Reply:
Yes. Not sure about the hardcoded path there tho; moreover we should be wary of using `/` as a path separator, I know there are some uses already but we should clean that up and certainly not add more cases, it will cause issues in OPAM 2.2 windows native builds.